fvm/src/gas/price_list.rs

```diff
@@ -703,13 +859,35 @@ impl PriceList {
     pub fn grow_memory_gas(&self, grow_memory_bytes: usize) -> Gas {
         self.wasm_rules.memory_expansion_per_byte_cost * grow_memory_bytes as i64
     }
+
+    #[inline]
+    pub fn on_actor_event(&self, evt: &ActorEvent) -> GasCharge {
```

Comment:
this follows with naming of the other gas methods, but knee-jerk reaction of this feels like its part of an event lib (maybe its my just JS showing though)

Reply:
:D Probably your JS history plays a major role in that bias here. It didn't even occur to me.